root/components/WithPager.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+import {ln} from '../static/scripts/common/i18n';
+import uriWith from '../utility/uriWith';
+import Frag from './Frag';
+import Paginator from './Paginator';
+import type {Node as ReactNode} from 'react';
+
+type Props = {|
+  +children: ReactNode,
+  +pager: PagerT,
+  +query: string,
+  +search?: boolean,
+  +total?: boolean,
+|};
+
+const WithPager = ({
```

Comment:
`WithPager` doesn't really match the rest of our component names. Simply Pager should do, no?

Reply:
You may be right, I was just copying root/components/with-pager.tt

I guess the idea was that it's not really a pager, it's wrapping content and *adding* pagers above/below it? But I think pager is a fake word in either case. :)

Reply:
Changed it to `PaginatedResults` if that sounds better.